test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -60,26 +58,46 @@ spec = do
         it "Wallet" $
             (eitherDecode exampleWallet :: Either String Wallet)
```

Comment:
Do we still need this?

Reply:
Nope. Feel free to remove, or I'll in the upcoming PR (also intending to use the roundtrip helpers provided by hspec-golden-aeson instead of the custom one we defined)

Reply:
I defined our own such that we can have custom settings, but not not pass this as an argument in
```haskell
roundtripAndGolden $ Proxy @ Wallet
```

Reply:
Removed (in first commit).